src/main/java/httpserver/Main.java

```diff
@@ -9,7 +17,11 @@ public static void main(String[] args) {
         HTTPServer httpServer = null;
             try {
                 SocketServer socketServer = new HTTPSocketServer(new ServerSocket(5000));
-                httpServer = new HTTPServer(5000, socketServer);
+                String path = "/Users/molliestephenson/Java/Server/cob_spec/public";
+                HTTPResourceHandler resourceHandler = new HTTPResourceHandler(path, new ResourceParser());
+                HTTPRequestHandler httpRequestHandler = new HTTPRequestHandler(new HTTPResponseBuilder(resourceHandler));
```

Comment:
I would prefer new HTTPesponseBuilder(resourceHandler) with its proper line, to make its more readible